### PR TITLE
[ALLUXIO-2266]Reference static method through the class in MetricsConfigTest

### PR DESCRIPTION
**Pull request by nicegdhj**
ALLUXIO-2266
Reference static method through the class in MetricsConfi…
https://alluxio.atlassian.net/browse/ALLUXIO-2266

### CODE REVIEW

**Comment by apc999** [comment]
ok to test

**Comment by apc999** [comment]
Jenkins, add to whitelist

**Comment by apc999** [comment]
/home/jenkins/workspace/Alluxio-Pull-Request-Builder/core/common/src/test/java/alluxio/metrics/MetricsConfigTest.java:65: error: Line is longer than 100 characters (found 106).

**Comment by apc999** [comment]
Builder/core/common/src/test/java/alluxio/metrics/MetricsConfigTest.java:65: error: Trailing whitespace found.
/home/jenkins/workspace/Alluxio-Pull-Request-Builder/core/common/src/test/java/alluxio/metrics/MetricsConfigTest.java:66: error: 'MetricsConfig' have incorrect indentation level 4, expected level should be 8.


You may want to run `mvn clean install -Dmaven.javadoc.skip=true -DskipTests` in your Alluxio source root to check the style before update the PR. thanks

File: core/common/src/test/java/alluxio/metrics/MetricsConfigTest.java [hunk 1]
```diff
@@ -62,7 +62,8 @@ public void subPropertiesTest() {
 
     Properties properties = config.getProperties();
 
-    Map<String, Properties> sinkProps = config.subProperties(properties, MetricsSystem.SINK_REGEX);
+    Map<String, Properties> sinkProps =
```
Comment (gpang) [inline core/common/src/test/java/alluxio/metrics/MetricsConfigTest.java]: I think there is a trailing whitespace at the end of the line.

**Comment by aaudiber** [comment]
alluxio-bot, check this please

File: core/common/src/test/java/alluxio/metrics/MetricsConfigTest.java (hunk 1 above)
Comment (nicegdhj) [inline core/common/src/test/java/alluxio/metrics/MetricsConfigTest.java]: thanks

**Comment by nicegdhj** [comment]
alluxio-bot, check this please

**Pull request merged**